src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdClientConfiguration.cs

```diff
@@ -98,7 +98,7 @@ public void Configure(string name, OpenIdConnectOptions options)
                 }
             }
 
-            if (settings.Parameters.Any())
+            if (settings.Parameters?.Any() == true)
```

Comment:
You need more C# 9 love in your life:

```suggestion
            if (settings.Parameters is { Length: > 0 })
```

Reply:
Would love to ;)

https://github.com/OrchardCMS/OrchardCore/blob/20d5554f78794d85acdb8b9e61a365f12b514b7b/src/OrchardCore.Build/OrchardCore.Commons.props#L9-L10

Reply:
Weird, I've never had any issue with that combination 😄 
Well, then... 👍🏻

Reply:
Maybe we could just use ;)

    <PropertyGroup>
      <LangVersion>9.0</LangVersion>
      <RazorLangVersion>Latest</RazorLangVersion>
    </PropertyGroup>

Will do a PR this night if it works

Reply:
Worth a crack @jtkech, super sad

Where it fails (for me - Seb had it failing on his perf tests - and found somewhere in the docs that it isn't supported) is if you try and use a C# 9 feature in a razor page when building with core 3